plugin/vscode/quick-lint-js/vscode-error-reporter.h

```diff
@@ -60,7 +60,22 @@ class vscode_error_formatter
             this->env_, reinterpret_cast<const char*>(this->message_.data())),
         /*severity=*/severity,
     });
-    diag.Set("code", ::Napi::String::New(this->env_, code.data(), code.size()));
+
+    char error_path[14];
+    sprintf(error_path, "/errors/%s", code.data());
```

Comment:
Must fix: Use `std::snprintf`, not `sprintf`. Also, `QLJS_ASSERT` that `sprintf`/`snprintf` didn't fail and that the result is not truncated.

Or, be lazy and use `std::string` concatenation.